packages/app/src/flame/transformFunction.ts

```diff
@@ -88,17 +116,17 @@ export function createFlameWgsl({
   }
 }
 
-export function extractFlameUniforms(flames: FlameFunction[]) {
-  const totalProbability = sum(flames.map((f) => f.probability))
+export function extractFlameUniforms(transforms: TransformFunction[]) {
```

Comment:
this should also use the ids I think. Otherwise we could get in trouble if the object order fails us.

Reply:
Refactored